web/experimental/new_embed.css

```diff
@@ -56,6 +56,13 @@ body {
 }
 
 
+/* Code Editor */
+
+#user-code-editor {
+  flex: 1;
```

Comment:
Does the Primer "flex-auto" class work here? If so, I'd prefer to stick with using the framework where possible.

https://primer.style/css/utilities/flexbox (no anchor link, unfortunately, so just Cmd-F to find flex-auto)

Reply:
It doesn't seem to work to use `flex 1 1 auto !important` with the `flex-basis` that our splitter is applying. I'm not sure why...

Reply:
Fair enough. Thanks for checking.